openquake/engine/db/schema/upgrades/0013-classical_damage.sql

```diff
@@ -0,0 +1,42 @@
+-- Damage curve
+CREATE TABLE riskr.damage (
+    id SERIAL PRIMARY KEY,
+    risk_calculation_id INTEGER NOT NULL REFERENCES uiapi.oq_job,
+    output_id INTEGER NOT NULL REFERENCES uiapi.output,
+    hazard_output_id INTEGER NULL REFERENCES uiapi.output,
```

Comment:
NULL is fine. This happens for statistics damage outputs which are not associated to any realization.